### PR TITLE
DEV-3388-new-go-exercise-display-p

### PR DESCRIPTION
**Pull request by jotapero**
PR was not created for this

### CODE REVIEW

File: solutions/onlyp/main.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
```
Comment (tiagocollot) [inline solutions/onlyp/main.go:2]: The folder name doesn't match the exercise name. Needs to be changed.

File: solutions/onlyp/main.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+import "github.com/01-edu/z01"
+
+func main() {
+	z01.PrintRune('P')
```
Comment (tiagocollot) [inline solutions/onlyp/main.go:6]: The instruction of the exercise doesn't match the solution. The instruction says that the output must be followed by a newline, our solution doesn't match it. Or we don't add a newline and remove it from the subject, or we add it and we fix the solution.

File: tests/onlyp_test/main.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package main
+
```
Comment (tiagocollot) [inline tests/onlyp_test/main.go:2]: The folder name doesn't match the exercise name. Needs to be changed.

File: tests/onlyp_test/main.go [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+import "github.com/01-edu/go-tests/lib/challenge"
+
+func main() {
+	challenge.Program("onlyP")
```
Comment (tiagocollot) [inline tests/onlyp_test/main.go:6]: Instead of "challenge.Program("onlyP")" should be ---> challenge.Program("displayp"). Needs to be changed.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: tests/onlyp_test/main.go [hunk 5]
```diff
@@ -0,0 +1,8 @@
+package main
+
+import "github.com/01-edu/go-tests/lib/challenge"
+
```
Comment (tiagocollot) [inline tests/onlyp_test/main.go:4]: We should increase the coverage of tests.

**Review by tiagocollot — CHANGES_REQUESTED** [review]

**Review by tiagocollot — APPROVED** [review]
LGTM!

File: solutions/displayp/main.go [hunk 6]
```diff
@@ -0,0 +1,8 @@
+package main
```
Comment (eslopfer) [inline solutions/displayp/main.go:1]: This file is no longer in the repo

File: tests/displayp_test/main.go [hunk 7]
```diff
@@ -0,0 +1,7 @@
+package main
```
Comment (eslopfer) [inline tests/displayp_test/main.go:1]: Likewise for this file

**Review by eslopfer — CHANGES_REQUESTED** [review]

**Pull request merged**